src/shared/components/magasinet-card/magasinet-card.css

```diff
@@ -0,0 +1,41 @@
+/**
+ * Block
+ */
+
+.hw-card-magasinet {
+  position: relative;
+  background: var(--hw-color-white);
+  padding: var(--hw-spacing-large) var(--hw-spacing-larger);
+  text-align: center;
+  border: 3px solid var(--hw-color-gray-light);
+
+&__overlay {
```

Comment:
hover should darken the card, like in the design. This can be done using the existing opaque colors in variables.css
Also overlay is not optional for this card. Should be implemented in an `:after`
Also, we need a transition for the hover effect.